content/blog/reprex-2-0-0/index.Rmd

```diff
@@ -0,0 +1,153 @@
+Note that this "current selection" default behaviour propagates to convenience wrappers around `reprex()`, such as `reprex_locale()` and venue-specific functions `reprex_r()`, as well as to the un-`reprex()` functions, such as `reprex_clean()`.
+
+[^2]: The `reprex_selection()` addin is alive and well!
+What's new is that it has become the default behaviour on RS/Cloud.
+`reprex_selection()` is so handy that many reprex users wire it up to a [custom keyboard shortcut](https://support.rstudio.com/hc/en-us/articles/206382178-Customizing-Keyboard-Shortcuts), such as Cmd + Shift + R (macOS) or Ctrl + Shift + R (Windows).
+
+Finally, in RS or on Cloud, the file containing the rendered reprex is opened and its contents are selected, ready for the user to copy via Cmd/Ctrl + C.
+
+These changes also make the ["Render reprex" addin/gadget](https://reprex.tidyverse.org/reference/reprex_addin.html) much more usable on RS and on Cloud. 
+
+## Working directory
+
+Ideally, a reprex is entirely self-contained and does not read any local files that, presumably, nobody else has.
+That's why, by default, `reprex()` works in an ephemeral directory created in the session temp directory.
+Run `getwd()` inside `reprex()`:
+
+```{r, eval = FALSE}
+reprex::reprex(
+  getwd()
+)
+```
+
+and you'll see something like this:
+
+``` r
+getwd()
+#> [1] "/private/tmp/RtmpXrUKXE/reprex-13063117cb6d8-alive-lark"
+```
+
+However, in the real world, sometimes you must read a local data file that you can't share with the world.
+You can still at least use `reprex()` to reveal your actual code and results.
+
+This has always been possible, but we've reworked some arguments to make this more natural to express:
+
+* `wd` is a new argument to set the reprex working directory
+* `outfile` is deprecated, with the existing `input` argument taking over its
+  duties
+
+If we run:
+
+```{r, eval = FALSE}
+reprex::reprex(
+  getwd(),
+  wd = "."
+)
+```
+
+we see the working directory is the directory where the source of this blog post lives:
+
+``` r
+getwd()
+#> [1] "/Users/jenny/rrr/tidyverse.org/content/blog/reprex-2-0-0"
+```
+
+The old way to `reprex()` in current working directory was `reprex(outfile =  NA)`, which was not very intuitive.
+You're not limited to `wd = "."`; you can use `wd` with any existing path.
+
+### Adjective-animal
+
+Various changes mean that more users will see reprex filepaths.
+Therefore, we've revised them to be more self-explanatory and human-friendly.
+When reprex needs to invent a file name, it is now based on a random "adjective-animal" slug.
+Bring on the `angry-hamster`! [^3]
+
+[^3]: `reprex()` draws randomly from a fixed set of adjective-animal slugs generated with the handy [ids package](https://reside-ic.github.io/ids/).
+
+You actually saw one of these filepaths already, in the output of `reprex(getwd())`:
+
+``` r
+getwd()
+#> [1] "/private/tmp/RtmpXrUKXE/reprex-13063117cb6d8-alive-lark"
+```
+
+## Local `.Rprofile` and renv happiness
```

Comment:
I think this section should follow the sectoin about `wd`, and be explained in that context.